uaa/src/main/resources/uaa.yml

```diff
@@ -553,6 +553,8 @@ uaa:
         - GET
         - HEAD
         - OPTIONS
+  features:
+#    aliasEntitiesEnabled: true
```

Comment:
I would omit this here, because if you want have it for documentation reasons you should add it in uaa-release

Then next question is: should be really call it features.
I would have expected a configuration option simply in login.xxxx name can be aliasEntitiesEnabled because it is a feature yes, but it should also help to configure behaviour... so I dont see this as a pure nice to have...
Many other configuration options in UAA are feature, but we dont use this term

Reply:
I think that having the `aliasEntitiesEnabled` setting (commented out) here as an example is a good thing. It will certainly help me if I want to check out the behavior when the setting is `true`.
Regarding calling it `features`, I am agreeing to @strehle. I think it can be called something specific instead.

Reply:
I also think calling this implementation as "feature flag" is not proper. When I first heard it today, I thought about IAM or role-based feature flag, which is turned on/off depending on who you are. However, here we have one configuration option that affects the whole instance.

Reply:
The feature flag terminology appears aligned with cloud foundry usage as in the docs: https://docs.cloudfoundry.org/adminguide/listing-feature-flags.html

Reply:
I moved the flag to `login`